polling/coincap.go

```diff
@@ -55,52 +57,45 @@ func CallCoinCap(config *config.Config) (CoinCapResponse, error) {
 	return CoinCapResponse, err
 }
 
-func HandleCoinCap(response CoinCapResponse, peg *PegAssets) {
+func HandleCoinCap(response CoinCapResponse, peg PegAssets) {
 
 	var timestamp = response.Timestamp
 
 	for _, currency := range response.Data {
-		if currency.Symbol == "XBT" || currency.Symbol == "BTC" {
-			value, err := strconv.ParseFloat(currency.PriceUSD, 64)
-			peg.XBT.Value = Round(value)
-			peg.XBT.When = timestamp
-			if err != nil {
-				continue
-			}
-		} else if currency.Symbol == "ETH" {
+		switch currency.Symbol {
+		case "BTC", "XBT":
 			value, err := strconv.ParseFloat(currency.PriceUSD, 64)
-			peg.ETH.Value = Round(value)
-			peg.ETH.When = timestamp
+			peg["XBT"] = PegItem{Value: Round(value), When: timestamp}
 			if err != nil {
 				continue
 			}
-		} else if currency.Symbol == "LTC" {
+		case "BCH", "XBC":
 			value, err := strconv.ParseFloat(currency.PriceUSD, 64)
-			peg.LTC.Value = Round(value)
-			peg.LTC.When = timestamp
+			peg["XBC"] = PegItem{Value: Round(value), When: timestamp}
 			if err != nil {
 				continue
 			}
-		} else if currency.Symbol == "XBC" || currency.Symbol == "BCH" {
+		case "ZCASH", "ZEC":
```

Comment:
Anyone prefer if we use ZEC or ZCASH?

Reply:
I think it should be ZEC, the ticker, to remain consistent with the rest.